src/patternfly/components/Table/table.scss

```diff
@@ -12,49 +12,57 @@
   --pf-c-table-caption--PaddingBottom: var(--pf-global--spacer--md);
   --pf-c-table-caption--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-table-caption--xl--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-table-caption--xl--PaddingLeft: var(--pf-global--spacer--md);
 
   @media screen and (max-width: $pf-global--breakpoint--xl) {
-    --pf-c-table-caption--PaddingRight: var(--pf-c-table-caption--md--PaddingRight);
+    --pf-c-table-caption--PaddingRight: var(--pf-c-table-caption--xl--PaddingRight);
     --pf-c-table-caption--PaddingLeft: var(--pf-c-table-caption--xl--PaddingLeft);
   }
 
   // Thead
-  --pf-c-table-thead--FontSize: var(--pf-global--FontSize--sm);
-  --pf-c-table-thead--FontWeight: var(--pf-global--FontWeight--bold);
-
-  // Thead cell
-  --pf-c-table-thead-cell--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-table-thead-cell--PaddingBottom: var(--pf-global--spacer--md);
-
-  // Cell
-  --pf-c-table-cell--hidden-visible--Display: table-cell;
+  --pf-c-table--thead--cell--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-table--thead--cell--FontWeight: var(--pf-global--FontWeight--bold);
 
-  // Td
-  --pf-c-table-tbody-cell--PaddingTop: var(--pf-global--spacer--lg);
-  --pf-c-table-tbody-cell--PaddingBottom: var(--pf-global--spacer--lg);
+  // Tbody cell
+  --pf-c-table--tbody--cell--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-table--tbody--cell--PaddingBottom: var(--pf-global--spacer--lg);
 
   // Th / td shared variables
-  // Padding vars must be individual as they are contextually reset
-  --pf-c-table-cell--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-table-cell--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-table-cell--PaddingBottom: var(--pf-global--spacer--md);
-  --pf-c-table-cell--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-table-cell--FontSize: var(--pf-global--FontSize--md);
-  --pf-c-table-cell--first-last-child--PaddingLeft: var(--pf-global--spacer--lg);
-  --pf-c-table-cell--first-last-child--PaddingRight: var(--pf-global--spacer--lg);
-  --pf-c-table-cell--first-last-child--xl--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-table-cell--first-last-child--xl--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-table--cell--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-table--cell--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-table--cell--Color: var(--pf-global--Color--100);
+  --pf-c-table--cell--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-table--cell--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-table--cell--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-table--cell--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-table--cell--first-last-child--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-table--cell--first-last-child--PaddingRight: var(--pf-global--spacer--lg);
+
+  // Default cell variables
+  --pf-c-table--cell--MinWidth: 0;
```

Comment:
did you set all these same vars in table-grid.scss ? could you use these vars in that file to keep consistent so if something changes here you dont have to update the grid file?

Reply:
Yes, I'm using these vars in `table-grid.scss`, but I'm unsetting them on the `th/td` and passing them to the child `.pf-c-table__text`. Otherwise, setting these values on the `th/td`s in grid would destroy the `pf-m-grid` layout. If there's no child `.pf-c-table__text`, then the table layout styles have no affect on content.